src/pages/QuestionPage.tsx

```diff
@@ -78,45 +154,54 @@ export default function QuestionPage() {
       justifyContent="center"
     >
       <Flex alignItems="center" maxW="md" margin="4">
-        {gotQuestion && prompt && answers && index ?
+        {prompt && answers && index ?
           <VStack>
             <Logo size={["32px", "50px"]} />
+            {isHost && <GameCode id={gameCode}></GameCode>}
+            <Timer index={index}/>
```

Comment:
I am a little sus about showing the timer on the host screen. It's possible for a player to join mid-way through a question and have 60 seconds to answer the question meanwhile the host screen is already done and the answer is shown.

I think we could go two routes with this:
1. Easy route. Keep this behaviour as is and turn it into a feature lol. No matter when a player joins, they will get 60s to answer the question (yay). This leaves a very large gaping hole for players to cheat by just lagging behind the host screen timer, but I guess that we just have to "encourage fair play".
2. Hard route. The backend keeps track of when a question is requested for the first time (aka when the host starts the question). We could even pass a key to the backend to let it know that this is the host requesting the question, and the timer should be started in the backend. After this point, whenever a question is requested from the backend, it comes with the time that is left on the question. This will ensure that the host and clients are somewhat aligned (there will likely be a difference of a few seconds).